packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useRef } from 'react'
+
+import { Pill } from '@/components/Pill'
+import { IconButton } from '@/components/_buttons/IconButton'
+import { SvgActionClose } from '@/components/_icons'
+
+import { ButtonsContainer, Tab, TabTitle, TabWrapper, TabsContainer, Topbar } from './DrawerHeader.styles'
+
+type TabsBarProps = {
+  title?: string
+  label: string
+  onCloseClick: () => void
+}
+
+export const DrawerHeader: React.FC<TabsBarProps> = React.memo(({ onCloseClick, title, label }) => {
+  const tabsContainerRef = useRef<HTMLDivElement>(null)
+  return (
+    <Topbar>
+      <TabsContainer ref={tabsContainerRef}>
```

Comment:
Do we really need three wrappers here? We have here TabsContainer, TabWrapper and Tab. Just asking out of curiosity. 
Also, I think it should be named TabContainer, not Tab**s**Container(without plural form)